src/Reader/Extension/PodcastIndex/Feed.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+/**
+ * @see       https://github.com/laminas/laminas-feed for the canonical source repository
+ * @copyright https://github.com/laminas/laminas-feed/blob/master/COPYRIGHT.md
+ * @license   https://github.com/laminas/laminas-feed/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Laminas\Feed\Reader\Extension\PodcastIndex;
+
+use Laminas\Feed\Reader\Extension;
+use stdClass;
+
+class Feed extends Extension\AbstractFeed
+{
+    /**
+     * Is the podcast locked (not available for indexing)?
+     *
+     * @return bool
+     */
+    public function isLocked()
+    {
+        if (isset($this->data['locked'])) {
+            return $this->data['locked'];
+        }
+
+        $locked = $this->xpath->evaluate('string(' . $this->getXpathPrefix() . '/podcast:locked)');
+
+        if (! $locked) {
+            $locked = false;
+        }
+
+        $this->data['locked'] = $locked === 'yes';
+
+        return $this->data['locked'];
+    }
+
+    /**
+     * Get the owner of the podcast (for verification)
+     *
+     * @return string
+     */
+    public function getLockOwner()
```

Comment:
Do not forget the `null` value.

```suggestion
    /**
     * Get the owner of the podcast (for verification)
     */
    public function getLockOwner(): ?string
```